scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate nodes with invalid categories."""
+
+import logging
+import sys
+
+from modularodm import Q
+from nose.tools import *
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+
+logger = logging.getLogger(__name__)
+
+
+#def main(dry=True):
+#    init_app(set_backends=True, routes=False, mfr=False)  # Sets the storage backends on all models
+#    do_migration(get_targets(), dry=dry)
+
+    
+def main():
+    print "in main"
+    # Set up storage backends
+    init_app(routes=False)
+    dry_run = 'dry' in sys.argv
+    if not dry_run:
+        scripts_utils.add_file_logger(logger, __file__)
+    logger.info("migrating personal to profileWebsites")
+            
+    for user in get_users_with_personal_websites():
+        logger.info(repr(user))
+        logger.info(repr(user.social))
+        if not user.social.get('profileWebsites'):
+            user.social['profileWebsites'] = [u'']
+            if user.social.get('personal'):
+                migrate_personal_to_profile_websites(user)
+        logger.info(repr(user.social))
+        if not dry_run:
+            user.save()
+
+
+def get_users_with_personal_websites():
+    return models.User.find(
+        Q('social', 'ne', None)
```

Comment:
The query should be constructed such that the script can be run more than once. If it is run more than once, a user should not have the same personal website appear twice in `profileWebsite`. This can currently happen, and is therefore a bug.